astropy/wcs/utils.py

```diff
@@ -126,6 +128,11 @@ def _celestial_frame_to_wcs_builtin(frame, projection='TAN'):
     elif isinstance(frame, Galactic):
         xcoord = 'GLON'
         ycoord = 'GLAT'
+    elif isinstance(frame, ITRS):
+        xcoord = 'TLON'
+        ycoord = 'TLAT'
+        wcs.wcs.radesys = 'ITRS'
+        wcs.wcs.date_obs = frame.obstime
```

Comment:
The attribute is `wcs.wcs.dateobs`. I think this will fail with `AttributeError`.

Reply:
Right you are! Fixed.